lib/configure-map.js

```diff
@@ -135,10 +135,24 @@ module.exports = function (original, length, options) {
 			return conf.delete(arg);
 		};
 	}
+	extGet = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return cache[id];
+	};
+	extHas = function () {
```

Comment:
Let's ensure that `extGet` and `extHas` have ensured right `lenght`, it can be done through `defineLength` as here -> https://github.com/epayet/memoizee/blob/16601d1d0762d34a844c9916cbe791b34c936aca/lib/configure-map.js#L123